javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var lightBulbClicker = document.querySelector('.light-bulb-on');
+var background = document.querySelector('.light-background');
+var i = true;
```

Comment:
How about a name like `lightOn`?

Reply:
You renamed the wrong thing. I was referring to `i`, the variable that gives no explanation of its purpose.